### ISSUE TITLE
Omg super...

### ISSUE THREAD

**Issue by karengrams** [karengrams/FinalesDeParadigmasDeProgramacion#8]
https://github.com/karengrams/FinalesDeParadigmasDeProgramacion/blob/c3ca92361c29dc6287ba92303f05d91f27b0dcf5/27%20de%20septimebre%20del%202017/270917.hs#L44-L46

Watch out! Super no compila...

**Comment by PabloOve** [comment]
Pero la idea esta bien nop?

**Comment by karengrams** [comment]
Yes. Pero el super no compilaria de la forma que vos lo pusiste.
